lib/sparql.jison

```diff
@@ -438,9 +478,9 @@ PN_LOCAL_ESC          "\\"("_"|"~"|"."|"-"|"!"|"$"|"&"|"'"|"("|")"|"*"|"+"|","|"
 "TO"                     return 'TO'
 "MOVE"                   return 'MOVE'
 "COPY"                   return 'COPY'
-"INSERT"\s+"DATA"        return 'INSERTDATA'
-"DELETE"\s+"DATA"        return 'DELETEDATA'
-"DELETE"\s+"WHERE"       return 'DELETEWHERE'
+"INSERT"{SPACES_COMMENTS}"DATA"  return 'INSERTDATA'
```

Comment:
What about

```sparql
INSERT # DATA
```
?

Reply:
Right, newlines aren't being checked properly atm.

Reply:
Do we leave this as a known TODO, or should this be fixed here?

Reply:
I already have a fix for it, push coming soon! :-)